src/overlays/actors/ovl_Bg_Hidan_Sekizou/z_bg_hidan_sekizou.h

```diff
@@ -6,9 +6,15 @@
 
 struct BgHidanSekizou;
 
+typedef void (*BgHidanSekizouUpdateFunc)(struct BgHidanSekizou*, GlobalContext*);
+
 typedef struct BgHidanSekizou {
-    /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x1C8];
+    /* 0x0000 */ DynaPolyActor dyna;
+    /* 0x0164 */ BgHidanSekizouUpdateFunc updateFunc;
+    /* 0x0168 */ s16 unk_168[4];
```

Comment:
```suggestion
    /* 0x0168 */ s16 timer[4];
    /* 0x0170 */ s16 flameSpeed;
```
0x168 is all generic timers, and 0x170 seems to be the speed at which the flames are expanding from the statues.